nemo_rl/models/policy/dtensor_policy_worker.py

```diff
@@ -418,6 +419,17 @@ def create_context_parallel_ctx(
 
     # Refer to nemo impl. Below is original comment.
     # based on https://github.com/pytorch/torchtitan/blob/cddd7dc809f36fe0ed51cdaaea0671c084d75442/torchtitan/distributed/utils.py#L178
+
+    def _apply_temperature_scaling(self, logits: torch.Tensor) -> torch.Tensor:
```

Comment:
kinda silly that we keep missing this. Can we modify a func test to use temp !=1 just to get some coverage?